packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -146,7 +155,7 @@ export class BlockList extends Component {
 							onCaretVerticalPositionChange={ this.onCaretVerticalPositionChange }
 							isSmallScreen={ ! this.props.isFullyBordered }
 						/> ) }
-					{ shouldShowInsertionPointAfter( clientId ) && <BlockInsertionPoint /> }
+					{ ! this.showGroupAppender() && shouldShowInsertionPointAfter( clientId ) && <BlockInsertionPoint /> }
```

Comment:
I lock render it here because it will be render in line 155 with `showSeparator` prop in `BlockListAppender`.